app/e2e/packetGroupPage.local.ts

```diff
@@ -0,0 +1,73 @@
+import { test, expect, Locator } from "@playwright/test";
+
+test.describe("Local packet group page", () => {
+  const packetId = "20240729-154652-95a6c08c";
+  let rows: Locator;
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./parameters");
+    const table = await page.getByRole("table");
+    await expect(table).toBeVisible();
+    rows = await table.getByRole("row");
+  });
+
+  // we can't assume new packets won't have been created, but should at least have those from the demo set
+  test("can see packet rows", async () => {
+    await expect(await rows.count()).toBeGreaterThanOrEqual(3);
+    const oldestRow = await rows.last();
+    const firstCell = await oldestRow.getByRole("cell").first();
+    const link = await firstCell.getByRole("link");
+    await expect(link).toHaveText(packetId);
+    await expect(await link.getAttribute("href")).toBe(`/parameters/${packetId}`);
+    const expectedDate = new Date(Date.UTC(2024, 6, 29, 15, 46, 52));
+    await expect(await firstCell.locator("div.text-muted-foreground")).toHaveText(expectedDate.toLocaleString());
+    const secondCell = (await oldestRow.getByRole("cell").all())[1];
+    const parameterPills = await secondCell.locator(".rounded-md").all();
+    await expect(parameterPills.length).toBe(3);
+    // Parameters are currently in a random order!
+    const parameterTexts = [];
+    for (const pill of parameterPills) {
+      parameterTexts.push(await pill.textContent());
+    }
+    parameterTexts.sort();
+    await expect(parameterTexts[0]).toBe("a:  1");
+    await expect(parameterTexts[1]).toBe("b:  2");
+    await expect(parameterTexts[2]).toBe("c:  3");
+  });
+
+  test("can search packet rows by id", async () => {
+    // Enter packet id in input
+    const packetInput = await rows.first().getByRole("cell").first().getByPlaceholder("Search...");
+    await packetInput.fill(packetId);
+    // should only have two rows left and last should have the id of the entered term
```

Comment:
Why two rows? I would expect a single row, for the one matching packet. So what is the first row for?

Reply:
OK, the first row is headers. Could you add a comment to say so please?